modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/random_sampler.yml

```diff
@@ -142,6 +142,62 @@ setup:
           }
   - match: { aggregations.sampled.mean.value: 1.0 }
 ---
+"Test random_sampler aggregation with scored subagg":
+  - requires:
+      cluster_features: ["gte_v8.2.0"]
+      reason: added in 8.2.0
```

Comment:
You will need to skip with a capability. This will allow testing with versions of 8.2.0, etc. and none of those versions have this bug fix.

Reply:
I was looking into that. (The BWC test fails obv.) How does that work? Where should I add a capability?

Reply:
I think I fixed it...